### ISSUE TITLE
2.8.2 Patch Release

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2948]
- #2900 
- #2901 
- #2782
- #2940
- #2949 
- #2945

## Pick List
```
git cherry-pick 88202ebf3e4aa6d8e7b3989f91179679d660a95b
git cherry-pick 583af497f703355ee062790e79802e6ea5492d33
git cherry-pick 3981483730df16715be55ced87aa66d35a840846
git cherry-pick e54469de6a8ae5a692fe669c6a14a30ce7058966
git cherry-pick 6f5d7b011453b46ef3557ff52cab7da980704558
git cherry-pick a71e277f87425f9e347c3dcd23055cdbdcb3cd16
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.8.1..HEAD --pretty='format:- %s'`
```
- Closes #2946 Video Thumbnail in Page/Event/News Editor Too Small (#2949)
- Closes #2944 Update style.css text field widths (#2945)
- Closes #2736: Remove whitespace on cards with no body or link. (#2940)
- Closes #2702: Add unpublished content layout to Marketing Cloud (#2782)
- Fixes #2888 Marketing Cloud Event Specificity (#2901)
- Closes #2102 Error upon save when linking to a document in a Card Link URL field (#2900)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.8.2
